pipelines/broad/dna_seq/germline/joint_genotyping/reblocking/ReblockGVCF.wdl

```diff
@@ -26,8 +26,8 @@ workflow ReblockGVCF {
 
   String gvcf_basename = basename(gvcf, gvcf_file_extension)
   # docker images
-  String gatk_docker_gcp = "us.gcr.io/broad-gatk/gatk:4.6.0.0"
-  String gatk_docker_azure = "terrapublic.azurecr.io/gatk:4.6.0.0"
+  String gatk_docker_gcp = "us.gcr.io/broad-gatk/gatk:4.6.1.0"
+  String gatk_docker_azure = "terrapublic.azurecr.io/gatk:4.6.1.0"
```

Comment:
Has this version of gatk been uploaded to azurecr.io? I believe Kevin did this for 4.6.0.0, and I'm not sure if we added this to the gatk release process yet or not. I think this was a todo still for the GATK team at the time of the 4.6.0.0 release.